### ISSUE TITLE
'Related Concepts' missing in aria-colspan and aria-rolespan attribute sections

### ISSUE THREAD

**Issue by devarshipant** [w3c/aria#2565]
**aria-colspan and aria-rolespan properties:** 

Corresponding html constructs like "colspan" and "rowspan" appear to be missing under the 'Characteristics' -> 'Related Concepts' section of the specification.

1. https://w3c.github.io/aria/#aria-colspan
2. https://w3c.github.io/aria/#aria-rowspan
